src/aap_eda/api/views/decision_environment.py

```diff
@@ -117,3 +109,26 @@ def retrieve(self, request, pk):
                 decision_environment.data
             ).data
         )
+
+    @extend_schema(
+        description="Delete a decision environment by id",
+        responses=status.HTTP_204_NO_CONTENT,
+    )
+    def destroy(self, request, *args, **kwargs):
+        instance = self.get_object()
+        force_delete = request.query_params.get("force", False)
```

Comment:
If force == "false", you will delete the object because `bool("false") == False` . You must to use string comparison and checking the positive case for safety: 
```suggestion
        force_delete = request.query_params.get("force", "false")
        ...
        if activations and force_delete.lower() != "true":
```

Reply:
on a second thought we might want to support as well "1" and "yes" so maybe it's better to write a helper to convert it. (I couldn't find any builtin tool for that)

Reply:
done.